src/manifestoo/main.py

```diff
@@ -266,6 +266,16 @@ def list_depends(
         "--as-pip-requirements",
         show_default=False,
     ),
+    sorting: str = typer.Option(
```

Comment:
Can you try if this does the validation?

```suggestion
    sorting: Literal["alphabetical", "topological"] = typer.Option(
```

Reply:
RuntimeError: Type not yet supported: typing.Literal['alphabetical', 'topological']

Even with the last version of typer.

Reply:
Ok, let's validate that the old way, then.